src/josepy/jws.py

```diff
@@ -57,22 +57,22 @@ class Header(json_util.JSONObjectWithFields):
     :ivar str cty: Content-Type, inc. :const:`MediaType.PREFIX`.
 
     """
-    alg = json_util.Field(
+    alg: jwa.JWASignature = json_util.field(
         'alg', decoder=jwa.JWASignature.from_json, omitempty=True)
-    jku = json_util.Field('jku', omitempty=True)
-    jwk = json_util.Field('jwk', decoder=jwk.JWK.from_json, omitempty=True)
-    kid = json_util.Field('kid', omitempty=True)
-    x5u = json_util.Field('x5u', omitempty=True)
-    x5c = json_util.Field('x5c', omitempty=True, default=())
-    x5t = json_util.Field(
+    jku: bytes = json_util.field('jku', omitempty=True)
+    jwk: jwk_mod.JWK = json_util.field('jwk', decoder=jwk_mod.JWK.from_json, omitempty=True)
+    kid: bytes = json_util.field('kid', omitempty=True)
+    x5u: bytes = json_util.field('x5u', omitempty=True)
+    x5c: bytes = json_util.field('x5c', omitempty=True, default=())
```

Comment:
Based on `x5c.{encoder,decoder}` below, I think this should be:
```suggestion
    x5c: Tuple[util.ComparableX509, ...] = json_util.field('x5c', omitempty=True, default=())
```
If you make this change, you'll also need to import `Tuple` from `typing`.

Reply:
Fixed that.